software/controller/lib/hal/hal_stm32.cpp

```diff
@@ -693,12 +339,19 @@ static void StepperISR() { StepMotor::DmaISR(); }
  *****************************************************************/
 
 // TODO: these could optionally call Fault() but not in production
```

Comment:
Well, even in production, they could if we make fault() buzz so the user knows something went wrong, and log something in EEPROM or Flash.

Reply:
Also added TODO note